static/src/javascripts-legacy/projects/commercial/modules/high-merch.js

```diff
@@ -2,13 +2,19 @@ define([
     'Promise',
     'common/utils/config',
     'common/utils/fastdom-promise',
+    'common/modules/experiments/ab',
     'common/modules/commercial/dfp/create-slot',
     'common/modules/commercial/commercial-features'
-], function (Promise, config, fastdom, createSlot, commercialFeatures) {
+], function (Promise, config, fastdom, ab, createSlot, commercialFeatures) {
     return {
         init: init
     };
 
+    function isLuckyBastard() {
```

Comment:
😝 hehe

...We should probably opt for something more abstract in case the variant appears in dotcom.feedback